config/block.block.drupalcampblockforlandingaboutevent_2018.yml

```diff
@@ -0,0 +1,22 @@
+uuid: a14dbe51-01d3-4f36-a006-6f61695ab368
```

Comment:
Just curious: how did you create these blocks?

Reply:
@juampynr 

DcampLandingAboutEventBlock is a custom block class. You can add this block to the theme and then export configuration. Default it will get the '_2' suffix, because there is already a block for 2017. I changed it to  2018 to make it more verbose.